prime-router/src/test/kotlin/azure/SettingFacadeTests.kt

```diff
@@ -245,11 +457,106 @@ class SettingFacadeTests {
         val orgJson = SettingsFacade(testMetadata(), accessSpy).findSettingsAsJson(OrganizationAPI::class.java)
         val listOrg = SettingsFacade(testMetadata(), accessSpy).organizations
         val org = SettingsFacade(testMetadata(), accessSpy).findOrganization("test")
-
+        val nameKV = "\"name\" : \"test\""
+        val stateCodeKV = "\"stateCode\" : \"CA\""
+        val jurisdictionKV = "\"jurisdiction\" : \"STATE\""
+        val descriptionKV = "\"description\" : \"Arizona PHD\""
+        val orgStr = mapper.writeValueAsString(listOrg)
         assertThat(listOrg.first().name).isEqualTo("test")
         assertThat(listOrg.first().description).isEqualTo(org?.description)
         assertThat(listOrg.first().jurisdiction).isEqualTo(Organization.Jurisdiction.STATE)
         assertThat(listOrg.first().stateCode).isEqualTo(org?.stateCode)
-        assertThat(orgJson).isEqualTo(mapper.writeValueAsString(listOrg))
+        assertTrue(orgJson.contains(nameKV))
+        assertTrue(orgStr.contains(nameKV))
+        assertTrue(orgJson.contains(stateCodeKV))
+        assertTrue(orgStr.contains(stateCodeKV))
+        assertTrue(orgJson.contains(jurisdictionKV))
+        assertTrue(orgStr.contains(jurisdictionKV))
+        assertTrue(orgJson.contains(descriptionKV))
+        assertTrue(orgStr.contains(descriptionKV))
+    }
+
+    @Test
+    fun `put org test happy case`() {
+        setupDatabaseAccess4PutTesting()
+        val (result, json) = SettingsFacade(testMetadata(), accessSpy).putSetting(
+            "waters",
+            orgJson,
+            AuthenticatedClaims.generateTestClaims(),
+            gov.cdc.prime.router.azure.OrganizationAPI::class.java,
+            null
+        )
+        assertTrue(result.name == "SUCCESS", "Expect SUCCESS as result, got ${result.name}")
```

Comment:
We should be using `assertThat` for tests

Reply:
done